src/App.jsx

```diff
@@ -6,14 +6,36 @@ function App() {
   const [monthlyContribution, setMonthlyContribution] = useState('')
   const [annualReturn, setAnnualReturn] = useState('7')
   const [yearsToRetirement, setYearsToRetirement] = useState('')
+  
+  // Real rate of return calculation inputs
+  const [nominalReturn, setNominalReturn] = useState('')
+  const [inflationRate, setInflationRate] = useState('')
+  
+  // Social security inputs
+  const [socialSecurityMonthly, setSocialSecurityMonthly] = useState('')
+  const [yearsUntilSocialSecurity, setYearsUntilSocialSecurity] = useState('')
```

Comment:
🔰 **Junior Developer**

Wow, this is so cool! I love how you added the Real Rate of Return calculator and social security features. 😁 Quick question though: for the `calculateRealReturn()` function (App.jsx line 16), what happens if someone enters a negative inflation rate? Should we handle that? Also, maybe add a little helper text to explain the formula to users? Just so it's not confusing!